app/frontend/components/info_popup.js

```diff
@@ -209,6 +223,25 @@ class InfoPopup {
     $('#info-popup-content').empty();
     $('#info-popup-content').html(appInfo.innerHTML);
     $('#app-info-tabs').tabs({ heightStyle: "fill" });
+
+    if (this.platformHelper.isElectron()) {
+      $('#export-user-data-button').bind('click', async () => {
+        var dialogTitle = i18n.t("general.export-user-data-action");
+        var filePath = await exportHelper.showSaveDialog('User_data_export', 'csv', dialogTitle);
+
+        await ipcDb.exportUserData(filePath);
+
+        $('#user-data-export-result').fadeIn();
+        await sleep(1500);
+        $('#user-data-export-result').fadeOut();
```

Comment:
Maybe the whole animation can be implemented with CSS animation triggered by applying certain class to it from here. 
But I need to think a little about it.

Reply:
Something for the future then?!